src/jdbc/cubrid/jdbc/jci/UConnection.java

```diff
@@ -2058,6 +2062,13 @@ public boolean protoVersionIsAbove(int ver) {
 		return false;
 	}
 
+	public static boolean protoVersionIsLower(int ver) {
+		if (protocolVersion < ver){
+			return true;
+		}
+		return false;
+	}
+
```

Comment:
there is protoVersionIsUnder(). why we need it?

Reply:
We need a new method as like protoVersionIsUnder in order to avoid side effect.
The readability could be considered by code refactoring in the future.